roles/tendrl-server/handlers/main.yml

```diff
@@ -35,3 +35,8 @@
   service:
     name=tendrl-notifier
     state=restarted
+
```

Comment:
Are we using this anywhere?

Reply:
I am using this to restart firewalld after enabling other ports

Reply:
ok

Reply:
Use reload instead of restart

Reply:
I need to recheck this.

Reply:
I need to recheck this.